rundeckapp/grails-app/controllers/rundeck/utils/UserDataTagLib.groovy

```diff
@@ -0,0 +1,34 @@
+package rundeck.utils
+
+import org.rundeck.app.data.providers.v1.UserDataProvider
+import org.rundeck.spi.data.DataManager
+
+class UserDataTagLib {
+    static namespace = "user"
+
+    UserDataProvider userDataProvider
+
+    def getJobFilters = { attrs, body ->
+        def jobFilters = userDataProvider.findByLogin(attrs.user)?.jobfilters
```

Comment:
There is no `getJobfilters()`, `getNodeFilters()` or `getReportFilters()` methods in the `RdUser` class, this doesn't seem to be correct

Reply:
I'm having a problem here, as `NodeFilter` and `ReportFilter` belongs to `User` but they aren't on the scope of this issue, that's why there are a lot of direct use of `User` as they are "ready" to be migrated. Do I include that migration on this PR?

Reply:
I think we will need to add method(s) to either the RdUser, or UserDataProvider, to enable "getJobFilters(username)", "getNodeFilters(username)", etc.
For Gorm provider, this can remain implemented via the domain classes, and any other provider implementation can choose to return empty result/throw Unimplimented if it is not available.  Then when using another provider we can use a feature flag to disable the features

Reply:
Let me know if the change I made is what you wanted

Reply:
for now we will just have a limitation in the SPI